api/common/api-profile.go

```diff
@@ -77,19 +78,24 @@ func AttributeToAPI(attrb kc.ProfileAttrbRepresentation, apiName string) *Profil
 		// Component should not be aware of its existence
 		return nil
 	}
-	// By default, attributes are not shown for a given frontend
+
+	// By default, attributes are not shown for a given frontend. Account is the only API supporting read-only
 	if apiName != "" && (!attrb.AnnotationMatches(apiName, func(value string) bool {
-		return strings.EqualFold(value, "true") || strings.EqualFold(value, "required")
+		return strings.EqualFold(value, "true") || strings.EqualFold(value, "required") || (apiName == "account" && strings.EqualFold(value, "read-only"))
```

Comment:
Not sure if the bridge should do this check... The bridge is a pass-through, it is the responsibility of the one doing the configuration and the one coding the self-service to do it correctly.